Tests/Form/Widget/FormSonataFilterChoiceWidgetTest.php

```diff
@@ -64,7 +63,7 @@ public function testDefaultValueRendering()
 
     protected function getParentClass()
     {
-        if (version_compare(Kernel::VERSION, '2.8.0', '>=')) {
+        if (class_exists('Symfony\Component\Form\Extension\Core\Type\RangeType')) {
```

Comment:
Let's try then.

Although  one of them reference to Kernel::VERSION >= 2.8.0 and the other one Kernel::VERSION >= 3.0